game/scripts/vscripts/components/progression/hero_progression.lua

```diff
@@ -1,48 +1,73 @@
 if HeroProgression == nil then
     HeroProgression = class({})
     Debug.EnabledModules['progression:*'] = true
+
```

Comment:
It's just what's in our editorconfig. Add the .editorconfig extension to atom and it'll just automatically confirm to most of our basic settings